src/bindings/system/fs.rs

```diff
@@ -110,6 +114,17 @@ pub fn init(lua: &Lua) -> ::Result<()> {
     Ok(())
 }
 
+#[cfg(target_os = "windows")]
+fn create_symlink(src_path: String, dest: String) -> std::io::Result<()> {
+    use std::os::windows::fs::symlink_file;
+    symlink_file(src_path, dest)
+}
+#[cfg(not(target_os = "windows"))]
```

Comment:
It might be best to get the target family vs the os since it would be more direct on what it is exactly defined as or related too vs just assuming its when its not windows. Eg `cfg(target_family = "windows")` and `cfg(target_family = "unix")`.